### ISSUE TITLE
Make star simulations 

### ISSUE THREAD

**Issue by tobias-liaudat** [aziz-ayed/denoising#1]
We need some basic star simulations to test the denoising methods.

This issue is to track the progress of this task. @aziz-ayed

**Comment by aziz-ayed** [comment]
I added the code for a simulation in the develop branch
It is formatted to work in Colab @tobias-liaudat

**Comment by tobias-liaudat** [comment]
Great, you could add some plots of some stars to check how do they look and verify they are ok.

**Comment by aziz-ayed** [comment]
Done, however the different stars look a lot like each other 
Probably we should take a bigger range for the ellipticities ? @tobias-liaudat

**Comment by tobias-liaudat** [comment]
@aziz-ayed  You should check what's the ellipticity of the stars you're plotting. Maybe is that the random ellipticity you're taking is close to zero.

But yes, seeing it again I think we can increase the ranges. We can use [-0.15, 0.15] for both ellipticities. The size range we can leave it as it is.

One thing, as we are not using a lot of stars you shouldn't use `np.random.uniform(-0.05, 0.05, 10)` but equally spaced values. Something like `np.linsapce(start=-0.15, stop=0.15, num=10, endpoint=True)`. There we can have a regular sampling of the hyper parameter space (e1, e2, R2) and be sure we have all the type of PSFs.

Another thing, maybe we can increase a little the noise. We can go for an SNR of 30.

**Comment by aziz-ayed** [comment]
Yes, I had plotted several couples of stars to be sure that the lookalike of the ellipticities wasn't due to two random picks having similar values but directly printing the ellipticities sure seems more natural !

Done the changes, also I plotted the first and the thousandth star. As we use linspace and itertools without shuffling, they should be the two stars with the biggest range between their ellipticities. 

Now the difference is pretty clear 🙂
